### DISCUSSION TITLE
How to delete extra square bracket at the end after nvim-cmp completion

### DISCUSSION THREAD

**Discussion by dil-mnasir** [LazyVim/LazyVim#1939] [Q&A]
I am using nvim-autopairs and disabled mini.pairs. When I am adding `[ngStyle]` its adding a an extra `]` at the end. 
How do I delete it automatically. 

https://github.com/LazyVim/LazyVim/assets/93813665/9c09a7c4-e2e2-4187-9821-3cba839be8e5

**Comment by dpetka2001** [comment]
This is similar to https://github.com/hrsh7th/nvim-cmp/issues/1532. See if you can ask there for how to solve your problem. The OP there seemed to fix his problem by simply updating the LSP server that he used.
